packages/jupyter-chat/src/model.ts

```diff
@@ -91,6 +92,11 @@ export interface IChatModel extends IDisposable {
    */
   readonly focusInputSignal?: ISignal<IChatModel, void>;
 
+  /**
+   * A signal emitting when the input attachments changed.
+   */
+  readonly inputAttachmentsChanges?: ISignal<IChatModel, IAttachment[]>;
```

Comment:
Maybe with the following spelling, similar to the names of other signals?

```suggestion
  readonly inputAttachmentsChanged?: ISignal<IChatModel, IAttachment[]>;
```